yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -389,33 +389,36 @@ Internet-Draft                 YANG Semver                 November 2021
 
 
 
-Clarke, et al.             Expires 6 May 2022                   [Page 7]
+Clarke, et al.             Expires 8 May 2022                   [Page 7]
 
 Internet-Draft                 YANG Semver                 November 2021
 
 
-           0.1.0
-             |
-           0.2.0
-             |
-           1.0.0
-             |  \
-             |   1.1.0 -> 1.1.1_compatible -> 1.1.2_non_compatible
-             |    |
-             |   1.2.0 -> 1.2.1_non_compatible -> 1.2.2_non_compatible
-             |    |
-             |   1.3.0 -> 1.3.1
-             |
-           2.0.0
-             |
-           3.0.0
-                \
-                 3.1.0
-
-   Assume the tree diagram above illustrates how an example YANG
-   module's version history might evolve.  For example, the tree might
-   represent the following changes, listed in chronological order of
-   when the revisions were published, from oldest revision to newest:
+            0.1.0
+              |
+            0.2.0
+              |
+            1.0.0
+              |
+            1.1.0 -> 1.1.1_compatible -> 1.1.2_non_compatible
+              |
+            1.2.0 -> 1.2.1_non_compatible -> 1.2.2_non_compatible
+              |  \
+            2.0.0 \
+              |    \--> 1.3.0 –> 1.3.1_non_compatible
+            3.0.0         |
+              |         1.4.0
+            3.1.0
+
+   The tree diagram above illustrates how the version history might
+   evolve for an example module.  The tree diagram only shows the
+   parent/child ancestry relationships between the revisions.  It does
```

Comment:
relationship?

Reply:
Slightly prefer plural - there are many relationships in the tree diagram, and a single module has multiple relationships.

Reply:
Plural reads well to me, too.